src/snowflake/snowpark/functions.py

```diff
@@ -1285,21 +1330,65 @@ def ltrim(e: ColumnOrName, trim_string: Optional[ColumnOrName] = None) -> Column
 
 def rpad(e: ColumnOrName, len: Union[Column, int], pad: ColumnOrName) -> Column:
     """Right-pads a string with characters from another string, or right-pads a
-    binary value with bytes from another binary value."""
+    binary value with bytes from another binary value.
```

Comment:
Could we add "`e` is padded to length `len` with characters/bytes from `pad`".

Reply:
added